src/frontend/src/index.ts

```diff
@@ -75,9 +74,13 @@ const init = async () => {
   // https://github.com/dfinity/internet-identity#build-features
   showWarningIfNecessary();
 
-  // Custom routing to the FAQ page
+  // Redirect to the FAQ
+  // The canister should already be handling this, this is just a safety
```

Comment:
can you improve the comment by saying that the canister retuns a redirect 301 for /faq?

I was confuse at first, added few review comments until I read the rust code

Reply:
Done, thanks! Added a comment in the canister as well